scipy/stats/_distn_infrastructure.py

```diff
@@ -2316,21 +2359,33 @@ def fit(self, data, *args, **kwds):
               function to be optimized) and ``disp=0`` to suppress
               output as keyword arguments.
 
+            - method : The method to use. The default is "MLE" (Maximum
+              Likelihood Estimate); "MM" (Method of Moments)
+              is also available.
+
+
         Returns
         -------
-        mle_tuple : tuple of floats
-            MLEs for any shape parameters (if applicable), followed by those
+        parameter_tuple : tuple of floats
+            Estimates for any shape parameters (if applicable), followed by those
             for location and scale. For most random variables, shape statistics
             will be returned, but there are exceptions (e.g. ``norm``).
 
         Notes
         -----
-        This fit is computed by maximizing a log-likelihood function, with
-        penalty applied for samples outside of range of the distribution. The
-        returned answer is not guaranteed to be the globally optimal MLE, it
+        With ``method="MLE"`` (default),
+        the fit is computed by maximizing a log-likelihood function, with
+        penalty applied for samples beyond the range of the distribution.
+
+        With ``method="MM"``, the fit is computed by minimizing the L2 norm
+        of the errors between the first *k* data moments and the corresponding
+        distribution moments, where *k* is the number of non-fixed parameters.
+        Typically, this error norm can be reduced to zero.
+
```

Comment:
```suggestion

For both methods, a penalty is incorporated into the objective function based on the number of observations outside the support of the distribution. An error is raised if optimization converges to fit parameters that are invalid, either because they are beyond the allowed range for the distribution or if they cause observations to be outside the support of the distribution.

```